### PR TITLE
Add a text to the page element

### PR DESCRIPTION
**Pull request by mldk-tech**
#117

### CODE REVIEW

**Comment by mldk-tech** [comment]
You can see that i add a text next to the youtube video.
ibli homepage

File: ibli/themes/custom/bootstrap_subtheme/sass/style.scss [hunk 1]
```diff
@@ -177,3 +177,16 @@ footer {
     overflow: inherit;
   }
 }
+
+#home-page-video {
```
Comment (NofarGizra) [inline ibli/themes/custom/bootstrap_subtheme/sass/style.scss]: Wrong file. These CSS lines should be located  in `_homepage.scss`, not is `style.scss`.

File: ibli/themes/custom/bootstrap_subtheme/sass/style.scss [hunk 2]
```diff
@@ -177,3 +177,16 @@ footer {
...
+  height: 315px;
+  width: 560px;
+  padding-left: 5px;
+  float: right;
+
+}
+
+#ibli-hom-page-paragraph {
```
Comment (NofarGizra) [inline ibli/themes/custom/bootstrap_subtheme/sass/style.scss]: Remove, this class is not in use.

**Comment by NofarGizra** [comment]
Merged.

**Pull request closed without merging**